src/main/java/com/onlinebookstore/controller/BookController.java

```diff
@@ -0,0 +1,39 @@
+package com.onlinebookstore.controller;
+
+import com.onlinebookstore.dto.BookDto;
+import com.onlinebookstore.dto.CreateBookRequestDto;
+import com.onlinebookstore.service.BookService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
+public class BookController {
+    private final BookService bookService;
+
+    @Autowired
```

Comment:
If you have only one constructor, don't need to use `@Autowired`, just use `@RequiredArgsConstructor`